jax_unirep/evotuning.py

```diff
@@ -1,27 +1,25 @@
 """API for evolutionary tuning."""
+import logging
```

Comment:
I like the use of logging here. It's more idiomatic than printing, even though printing is more commonly used.